src/cli/site/deploys.js

```diff
@@ -0,0 +1,68 @@
+const chalk = require('chalk')
+const ora = require('ora')
+const columnify = require('columnify')
+
+const { list } = require('../../api/deploy')
+const { humaniseError } = require('../../utils/api-error')
+const { options } = require('../../utils/cli')
+
+const main = async args => {
+  let index
+  let spinner
+  if (!args.json) {
+    spinner = ora('Connecting to Calibre')
+    spinner.color = 'magenta'
+    spinner.start()
+  }
+
+  try {
+    index = await list(args)
+    if (args.json) return console.log(JSON.stringify(index, null, 2))
+  } catch (e) {
+    if (args.json) return console.error(e)
+    spinner.fail()
+    throw new Error(humaniseError(e))
+  }
+
+  spinner.stop()
+  console.log(`${chalk.bold(index.deploys.length)} deploys`)
+
+  console.log(
+    columnify(index.deploys, {
+      columnSplitter: ' | ',
+      truncate: true,
+      maxLineWidth: 'auto'
+    })
+  )
+
+  console.log(
+    '\n',
+    columnify(
+      [
+        {
+          'More?': index.pageInfo.hasNextPage,
```

Comment:
Is this just for debugging/development?

Reply:
Perhaps we could figure out the command that they need to run in order to get the next page, then supply that? 

eg: 

> To see deploys prior to release `djd301`, run `calibre site deploy-list --site=calibre --cursor="M2o="`

Reply:
No, this outputs if there's more pages and what the cursor is. e.g.

```
MORE? | CURSOR
true  | asd234jkln
```

It does feel a little clunky …

Reply:
Yeah, I just had the same thought! Will update.